### DISCUSSION TITLE
http连接没有echo？

### DISCUSSION THREAD

**Discussion by new3Git** [Mrs4s/go-cqhttp#1622] [Q&A]
http://127.0.0.1:8700/get_msg?message_id=-1403786170&echo=%221221acc%22
无论echo是否加引号，最终返回值都是这个

{"data":{"group":false,"message":"[CQ:face,id=179]","message_id":-1403786170,"message_id_v2":"00000000a432353d000021e9","message_seq":8681,"message_type":"private","real_id":8681,"sender":{"nickname":"name","user_id":xxxxxx},"time":1660205213},"retcode":0,"status":"ok"}

配置文件仅更改监听地址，其他都是默认

**Comment by Akegarasu** [comment]
http没有
